### ISSUE TITLE
Use semantic import versioning

### ISSUE THREAD

**Issue by refs** [cs3org/reva#279]
[Semantic Import Versioning](https://github.com/golang/go/wiki/Modules#semantic-import-versioning) might help with import names

**Comment by labkode** [comment]
@refs for the scope of internal reva packages or for the go-cs3apis?

**Comment by refs** [comment]
In the go-cs3apis scope

**Comment by labkode** [comment]
@refs definitely, once the APIs are stable enough (yesterday I changed like 4 times)  we can start tagging them
